addons/web/static/tests/views/list_view_tests.js

```diff
@@ -4941,6 +4942,48 @@ QUnit.module("Views", (hooks) => {
         assert.containsN(target, "tbody td.o_list_record_selector", 3, "should have 3 records");
     });
 
+    QUnit.test("custom delete confirmation dialog", async (assert) => {
+
+        class CautiousController extends ListController {
+            get deleteConfirmationDialogProps() {
+                let props = super.deleteConfirmationDialogProps;
```

Comment:
const props

Reply:
I actually prefer `let` since it indicates it will be changed... but since you're the codeowner... :)